cookiecutter.json

```
@@ -3,6 +3,5 @@
   "repository_hosting_platform": ["GitHub", "GitLab"],
   "organisation_handle": "Your GitHub/GitLab organisation name, for example ukgovdatascience",
   "organisational_framework": ["GDS", "N/A"],
-  "project_name": "Your new project name",
-  "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}"
+  "project_name": "Your new project name"
```

Happy with this, I think it might make sense to try and push people towards using lowecases and underscores/dashes in the default argument (doesn't really matter as package name handles this!)
```suggestion
  "project_name": "your_new_project_name"
```